sdk/appconfiguration/app-configuration/samples/react-sample/src/components/Page.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+*/
+
+import React from "react";
+import { useEffect, useState } from "react";
+import {
+  Navbar,
+  Nav,
+  NavDropdown,
+  Jumbotron,
+  Alert,
+  Row,
+  Col,
+  Container
+} from "react-bootstrap";
+import { getEnvironmentVariable } from "../utils";
+import {
+  AppConfigurationClient,
+  featureFlagPrefix,
+  isFeatureFlag,
+  isFeatureFlagClientFilter
+} from "@azure/app-configuration";
+
+let client: AppConfigurationClient;
+
+const featureFlag1 = {
+  id: "react-app-feature-1",
+  description: "",
+  enabled: false,
+  conditions: {
+    client_filters: []
+  }
+};
+const featureFlag2 = {
+  id: "react-app-feature-2",
+  description: "",
+  enabled: true,
+  conditions: {
+    client_filters: [
+      {
+        name: "Microsoft.TimeWindow",
+        parameters: {
+          Start: "Mon, 12 Apr 2021 07:00:00 GMT",
+          End: "Wed, 14 Apr 2021 07:00:00 GMT"
+        }
+      }
+    ]
+  }
+};
+
+export default function Page(): JSX.Element {
+  const feature1 = "react-app-feature-1";
+  const feature2 = "react-app-feature-2";
+  const [feature1ClassName, setFeature1ClassName] = useState<string>("feature-button-hidden");
+  const [feature2ClassName, setFeature2ClassName] = useState<string>("feature-button-hidden");
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What dependencies are you missing that raised this error? It's not _too_ important for a sample honestly but would be nice to be exhaustive if possible. It might be something transitive  (useEffect -> getFeatureFlags -> ???)

Reply:
Removed this dependency.